index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>My Website</title>
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="icon" href="./favicon.ico" type="image/x-icon">
+  </head>
+  <body>
+
+  <!-- HEADER SECTION (major section) -->
+
+  <header id="headerSection">
+
+  <!-- navigation area (subsection) -->
+    <nav id="navBar">
+      <div id="logoContainer" class="navitems">
+        <p id="logo">My Logo</p>
+      </div>
+
+      <div id="menuiconContainer" class="navItems">
+      	<img src="./media-library/Union.png" alt="menu" width="" height="">
+      </div>
+
+    </nav>
+
+    <!-- about area (subsection) -->
+    <section id="aboutContainer">
+      <div id="aboutTitle">
+        <p>I’m Alexandra. Glad to see you!</p> 
+      </div>
+
+      <div id="aboutSummary">
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+      </div>
+
+      <div id="socialsContainer">
+      <p>LET’S CONNECT</p>
+      <img class="socialIcons" src="./media-library/Vector-4.png" alt="" width="" height="">
+      <img class="socialIcons" src="./media-library/Vector-3.png" alt="" width="" height="">
+      <img class="socialIcons" src="./media-library/Vector-2.png" alt="" width="" height="">
+      <img class="socialIcons" src="./media-library/Vector-1.png" alt="" width="" height="">
+      <img class="socialIcons" src="./media-library/Vector.png" alt="" width="" height="">
+      </div>
```

Comment:
- Kindly for the social media links use a `ul `tag to group them, and a` li `per icon; and make sure each icon has an` a` link (a tag). Apart from being semantically correct, a tag will let you connect your social networks with the icons, using href :+1: 

```
 <ul>
      <li><a href=""><img src="./images/twitter.png" alt=""></a></li>
      <!-- other links here -->
 </ul>
```